tests/php/Extension/FluentFilteredExtensionTest.php

```diff
@@ -45,14 +43,14 @@ protected function setUp(): void
     public function testAugmentSQLFrontend()
     {
         // Specifically set this config value so that filtered locales ARE required in stage=Stage.
-        Config::modify()->set(DataObject::class, 'apply_filtered_locales_to_stage', true);
+        SiteTree::config()->set('apply_filtered_locales_to_stage', true);
```

Comment:
Why is this swapping from being applied to `DataObject` to now being applied to `SiteTree` specifically?

Reply:
Same question for other changes blow.

Reply:
The test is for `SiteTree`, hence I used `SiteTree` config. Functionally doesn't matter, it just makes the config scope change fit the test.